src/gauss_pdf_and_cdf.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import plotly.graph_objects as go
+from sympy import Symbol, pi, E, sqrt, integrate, oo
+from abc import ABC, abstractmethod
+from scipy import stats
+import math
+
+
+class ProbabilisticFunction(ABC):
+    @abstractmethod
+    def __init__(self):
+        pass
+
+
+class Visualizer(ProbabilisticFunction):
```

Comment:
visualizer must not inherit ProbabilisticFunction because it's not pdf itself, it's used in pdf.